### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by jseppi** [TNRIS/iswp2017#145]
Accessibility and security review from IT

**Comment by jseppi** [comment]
Initial request sent to IT on 1/29

**Comment by jseppi** [comment]
Initial review feedback:

We determined that you’ll need to request an exception for this site.  With approved exception, the site is exempted from accessibility requirements.  The reason for the exception is that the site is using maps which is generally not accessible and can’t be made accessible.

There are some technical issues that need to be addressed:

* Skip to Main Content does not work correctly – make sure the intended content target is set properly.
* View data for combo boxes needs to have defined ARIA live regions.
* Add a button to update the page after user selects the combo box items (region/types) to help users to expect changes in the page.
* The “View Data” button is not very clear, consider something more descriptive.
* The hidden tabular information in the chart is hidden from screen readers, consider using ARIA live region or always show tabular data.
* In the table, since it is using <th> header, it is recommended to have scope defined with <th scope=”col”> and <th scope=”row”>.
* Blue/white bars under Demands by Usage Type (and other areas) are tabbable, but has no indication that you’re tabbed in.

**Comment by jseppi** [comment]
> My recommendation is for the Skip to Main Content to go to the combo boxes to view the data.  Right now, it will go directly to the map and covers up the combo boxes.

**Comment by jseppi** [comment]
> add an alt attribute to indicate that the home button is indeed home.  Some screen readers (depending on an individual configuration) will not read title attributes.

**Comment by jseppi** [comment]
Closed in 266703ff00fb852331780bca3d94bb7b67f8d1ae
